dlme_airflow/drivers/iiif_json.py

```diff
@@ -28,8 +28,18 @@ def _open_collection(self):
         resp = self._get(self.collection_url)
         if resp.status_code == 200:
             collection_result = resp.json()
-            for manifest in collection_result.get("manifests", []):
-                self._manifest_urls.append(manifest.get("@id"))
+            try:
+                for manifest in collection_result["manifests"]:  # IIIF version 2
+                    try:
+                        self._manifest_urls.append(manifest["@id"])
+                    except KeyError:
+                        self._manifest_urls.append(manifest.get("id"))
+            except KeyError:
+                for manifest in collection_result.get("items", []):  # IIIF version 3
+                    try:
+                        self._manifest_urls.append(manifest["@id"])
+                    except KeyError:
+                        self._manifest_urls.append(manifest.get("id"))
```

Comment:
Instead of using exception handling here I wonder if it makes sense to first look for the manifests and then iterate over them? So something like:

```python
if 'manifests' in collection_result: # IIIF v2
    manifests = collection_result['manifests']
elif 'items' in collection_result: # IIIF v3
     manifests = collection_result['items'] 
else:
     raise Exception(f"Unknown collection manifest format: {self.collection_url}")

for manifest in manifests:
    if '@id' in manifest:
        url = manifest['@id'] # IIIF v2?
    elif 'id' in manifest:
        url = manifest['id'] # IIIF v3
    else:
        raise Exception(f'Unknown URL in manifest: {manifest}')
    self._manifest_urls.append(url)
```

And maybe we should add a test?